ansible/playbooks/AdoptOpenJDK_AIX_Playbook/roles/yum/tasks/main.yml

```diff
@@ -94,9 +94,6 @@
 - name: Install yum package support
   yum: name={{ item }} state=present update_cache=yes
   with_items:
-    - http://www.bullfreeware.com/download/bin/2328/libiconv-1.14-22.aix6.1.ppc.rpm
-    - http://www.bullfreeware.com/download/bin/2591/libunistring-0.9.6-2.aix6.1.ppc.rpm
-    - http://www.bullfreeware.com/download/bin/3944/perl-5.24.0-3.aix6.1.ppc.rpm
     - http://www.oss4aix.org/download/RPMS/cmake/cmake-3.7.2-1.aix6.1.ppc.rpm
   tags: rpm_install
```

Comment:
Could you add yum tags to this task and the one below it please. The proper syntax is 
```
  tags: 
    - rpm_install
    - yum
```